python/ray/exceptions.py

```diff
@@ -16,6 +16,13 @@ class RayConnectionError(RayError):
     pass
 
 
+class RayCancellationError(RayError):
+    """Raised when this task or a dependency is cancelled"""
+
+    def __str__(self):
+        return "This task was cancelled"
```

Comment:
Can we have a better description? (like what was the function that invokes this task)?

Reply:
Do you mean in the string representation?